about.html

```diff
@@ -0,0 +1,108 @@
+        </div>
+    </div>
+    <div class="main-program">
+        <div class="div-list">
+            <div class="item-list">
+                <img src="./images/list2.png" alt="conversation" class="images">
+                <h3 class="img-title">Literary Talent</h3>
+                <p class="img-des">Among the representatives of Spain, David Roas, a Spanish writer and literary critic, specializing in fantastic literature, stands out, who will share dialogue tables 
+                    on literary criticism and crime novels. Once again, we are visited by Juan Casamayor, founder of Páginas de Espuma, an independent label that is a reference in the genre of short stories in Spanish.</p>
+            </div>
+        </div>
+        <div class="space"></div>
+    </div>
+    <div  class="featured-speakers">
+        <h2 class="speakers-title">More about the event</h2>
+        <div class="linea-2"></div>
+        <h3 class="guion">if you want more information about the event go here:</h3>
+        <div class="other-programs">
+            <a href="https://elperuano.pe/noticia/163398-atencion-lectores-en-julio-vuelve-la-feria-internacional-del-libro-de-lima#:~:text=La%20espera%20termin%C3%B3%3A%20la%2026%C2%AA,avenida%20Salaverry%2C%20en%20Jes%C3%BAs%20Mar%C3%ADa." class="program">
+                <img src="./images/feria.jpg" alt="feria" class="feria-img">
+                <span class="cover"></span>
+                <h3 class="year">2022</h3>
+                <p class="sentence">26° Book fair</p>
+            </a>
+            <a href="https://librosami.pe/2019/06/feria-internacional-del-libro-de-lima-2019-autores-invitados-y-todo-lo-que-debes-saber-del-evento/" class="program">
+                <img src="./images/feria2.jpg" alt="feria" class="feria-img">
+                <span class="cover"></span>
+                <h3 class="year">2019</h3>
+                <p class="sentence">24° Book fair</p>
+            </a>
+        </div>
+    </div>
+    <div class="div-sponsor">
+        <h2 class="sponsor-title">Sponsored by</h2>
+        <div class="linea"></div>
+        <ul class="sponsor">
+            <li class="sponsor-img"><img src="images/el-comercio.png" alt="el-comercio" class="sponsor-img"></li>
+            <li class="sponsor-img"><img src="images/cato2.png" alt="catolica" class="sponsor-img"></li>
+            <li class="sponsor-img"><img src="images/rpp.png" alt="rpp-noticias" class="sponsor-img"></li>
+            <li class="sponsor-img"><img src="images/tvperu.png" alt="tv-Perú" class="sponsor-img"></li>
+            <li class="sponsor-img"><img src="images/bbva2.png" alt="bbva" class="sponsor-img"></li>
+        </ul>
+    </div>
+    <footer class="site-footer">
+        <div class="footer">
+            <h2 class="slogan"><i class="fa-solid fa-cubes"></i> J.B. Desing</h2>
+            <p class="copyright"><i class="fa-solid fa-copyright"> project make by Josue Benavides some Rights Reserved</i></p>
+        </div>
+    </footer>
```

Comment:
- [ ] Please follow the layout: composition and space between elements.
- [ ] In the mobile view partner layout changing and it is two lines. Your design is still one line. Kindly change it.

### YOUR DESIGN MOBILE VIEW
![image](https://user-images.githubusercontent.com/38635158/176970856-3adc2048-4dd8-4b4e-a6e8-dbc0e03e76ce.png)
____
### ORIGINAL MOBILE VIEW
![image](https://user-images.githubusercontent.com/38635158/176969996-8612d0cf-c43a-46b1-a255-0c7fd2c75349.png)